sqlglot/executor/python.py

```diff
@@ -150,7 +150,8 @@ def scan_csv(self, step):
                             types.append(type(ast.literal_eval(v)))
                         except (ValueError, SyntaxError):
                             types.append(str)
-                context.set_row(tuple(t(v) for t, v in zip(types, row)))
+                # handle '' in csv
+                context.set_row(tuple(t(v) if v else 0 for t, v in zip(types, row)))
```

Comment:
What triggered the need for this change?

Reply:
I ran into this error `invalid literal for int() with base 10: ''`, as in the tpc-ds dataset there some null values

Reply:
that probably means there's another bug so don't do that